regression/goto-instrument/enum-range-check1/main.c

```diff
@@ -1,23 +1,26 @@
 #include <stdio.h>
 
-enum day {sunday = 1,
-          monday,
-          tuesday = 5, 
-          wednesday,
-          thursday = 10,
-          friday,
-          saturday}; 
+enum day
```

Comment:
Apply clang-format as you go (e.g. do a rebase with `git-clang-format HEAD~1` after each commit)

Reply:
Thanks for the advice, should I squash the clang-format commit with the rest now?

Reply:
I guess once the others are incrementally formatted then it won't exist at all